index.html

```diff
@@ -81,9 +81,191 @@ <h1>
         </div>
       </div>
     </section>
-    <section class="cards">
-      <div class="card">
-        <div class="card__content flex__container">
+    <section id="portfolio" class="cards mobile">
+      <div class="card project-mobile">
+        <div class="card__content">
+          <img
+            src="images/Snapshoot-Portfolio.png"
+            alt=""
+            class="card__image"
+          />
```

Comment:
It was a great job, we can notice that images in this section are missing the alt attribute, I think it will be better to fix that for better accessibility.